pkg/internal/checkup/executor/console/console.go

```diff
@@ -45,24 +52,24 @@ const (
 func NewExpecter(serialConsoleClient vmiSerialConsoleClient,
 	vmiNamespace,
 	vmiName string,
-	timeout time.Duration,
-	opts ...expect.Option) (expect.Expecter, <-chan error, error) {
-	return spawnConsole(serialConsoleClient, vmiNamespace, vmiName, timeout, opts...)
+	opts ...expect.Option) Expecter {
+	return Expecter{
+		serialConsoleClient: serialConsoleClient,
+		vmiNamespace:        vmiNamespace,
+		vmiName:             vmiName,
+		opts:                opts,
+	}
 }
 
-func spawnConsole(serialConsoleClient vmiSerialConsoleClient,
-	vmiNamespace,
-	vmiName string,
-	timeout time.Duration,
-	opts ...expect.Option) (*expect.GExpect, <-chan error, error) {
+func (e Expecter) spawnConsole(timeout time.Duration) (*expect.GExpect, error) {
```

Comment:
Why was the error channel removed?
What are the consequences of the removal?

Reply:
> Why was the error channel removed? 

it wasn't used.

> What are the consequences of the removal?

none

Reply:
the was complaining about returning the <-chan error, because it is not really used anywhere.. removing it fixed the linter.